lib/tether/tether.go

```diff
@@ -470,6 +470,8 @@ func (t *tether) Start() error {
 			return err
 		}
 
+		// Danger, Will Robinson! There is a strict ordering here.
```

Comment:
Unfortunately t.extensions is a map - with no ordering guarantees. What is the impact of this ordering being incorrect?
We cannot merge this unless either the comment is wrong, we resolve the dependency, or we enforce the ordering.